Sources/Rendering/OpenGL/PolyDataMapper/index.js

```diff
@@ -1236,6 +1236,28 @@ function vtkOpenGLPolyDataMapper(publicAPI, model) {
       } else {
         cellBO.getVAO().removeAttributeArray('normalMC');
       }
+
+      model.renderable.getCustomShaderAttributes().forEach((attrName, idx) => {
+        if (cellBO.getProgram().isAttributeUsed(`${attrName}MC`)) {
+          if (
+            !cellBO
+              .getVAO()
+              .addAttributeArray(
+                cellBO.getProgram(),
+                cellBO.getCABO(),
+                `${attrName}MC`,
+                cellBO.getCABO().getCustomData()[idx].offset,
+                cellBO.getCABO().getStride(),
+                model.context.FLOAT,
+                3,
+                false
```

Comment:
Is that 3 there correct? Maybe cellBO.getCABO().getCustomData()[idx].components

Reply:
correct! thanks